sigma/plugins/utility/other/define.py

```diff
@@ -35,12 +35,12 @@
 
     try:    
         if '<span class="intro-colon">' not in str(soupObj):
-            raise IndexError('The website has changed, please fix. ("intro-colon")')
+            raise ValueError('The website has changed, please fix. ("intro-colon")')
```

Comment:
You can change this to just the generic Exception instead. Value nor Index errors make sense in this.